CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 
+### Fixed
+
+- Fix a bug where bad responses would be cached
+
```

Comment:
Can you move this up to `[unreleased]` since [1.1.0](https://github.com/rokucommunity/roku-requests/releases/tag/v1.1.0) has already been published?